.circleci/config.yml

```diff
@@ -1,112 +1,49 @@
 version: 2.1
-
+setup: true # Use https://circleci.com/docs/dynamic-config/ to trigger the `test-deploy` workflow.
 orbs:
-  # Replace this with your own!
-  synthetics-ci-orb: datadog/synthetics-ci-orb@<<pipeline.parameters.dev-orb-version>>
-  orb-tools: circleci/orb-tools@10.0
+  orb-tools: circleci/orb-tools@11.5
   bats: circleci/bats@1.0
   shellcheck: circleci/shellcheck@2.0
 
-# Pipeline Parameters
-## These parameters are used internally by orb-tools. Skip to the Jobs section.
-parameters:
-  run-integration-tests:
-    description: An internal flag to prevent integration test from running before a development version has been created.
-    type: boolean
-    default: false
-  dev-orb-version:
-    description: >
-      The development version of the orb to test.
-      This value is automatically adjusted by the "trigger-integration-tests-workflow" job to correspond with the specific version created by the commit and should not be edited.
-      A "dev:alpha" version must exist for the initial pipeline run.
-    type: string
-    default: "dev:alpha"
-
-jobs:
-  # Define one or more jobs which will utilize your orb's commands and parameters to validate your changes.
-  integration-test-simple:
-    docker:
-      - image: cimg/base:stable
-    steps:
-      - checkout
-      - synthetics-ci-orb/run-tests:
-          fail_on_critical_errors: true
-  integration-test-with-spaces:
-    docker:
-      - image: cimg/base:stable
-    steps:
-      - checkout
-      - synthetics-ci-orb/run-tests:
-          fail_on_critical_errors: true
-          files: "ci/file\ with\ space.json"
-  integration-test-with-multiple-public-ids:
-    docker:
-      - image: cimg/base:stable
-    steps:
-      - checkout
-      - synthetics-ci-orb/run-tests:
-          fail_on_critical_errors: true
-          public_ids: 6jq-89z-5ur, s26-j8m-bu7
+filters: &all-branches-and-tags
+  tags:
+    only: /.*/
```

Comment:
This is a YAML alias that we reuse in the jobs (with `*all-branches-and-tags`) that we want to run on every commit and tag.

> CircleCI does not run workflows for tags unless you explicitly specify tag filters. Additionally, if a job requires any other jobs (directly or indirectly), you must [use regular expressions](https://circleci.com/docs/workflows/#using-regular-expressions-to-filter-tags-and-branches) to specify tag filters for those jobs. Both lightweight and annotated tags are supported. [Source](https://circleci.com/docs/workflows/#executing-workflows-for-a-git-tag)